### ISSUE TITLE
Random search

### ISSUE THREAD

**Issue by sari777** [vladkens/ecloop#3]
you should done it in random search. not sequential.
also not hitting in the specifc range hash160 in sequential.(in previous version it was working)

**Comment by vladkens** [comment]
@sari777 any example?

**Comment by vladkens** [comment]
@sari777 random search added in version. Check [readme](https://github.com/vladkens/ecloop?tab=readme-ov-file#random-search) for usage example.

Reopen this issues or open new if you have any questions.

**Comment by sari777** [comment]
> [@sari777](https://github.com/sari777) random search added in version. Check [readme](https://github.com/vladkens/ecloop?tab=readme-ov-file#random-search) for usage example.
> 
> Reopen this issues or open new if you have any questions.

dear, random function is not working . not hitting the private key even within 24 bit range. and what is the purpose of bloom filter. ?

**Comment by vladkens** [comment]
@sari777 I can't understand how you are trying to use Random Search.

Random Search checks priv keys in a random range, calculates pubkey (compressed, uncompressed) -> hash160 from them. Then it checks if hash160 is in the filter (if yes prints it to console). The filter can be either a list of hash160 or a bloom filter for speed.

**Comment by sari777** [comment]
yes. random search check priv keys in given random range. i believe. but its not hitting the private key in rnd function. even i choose small range. like 13-17 bits. thats all i am saying.
